### PR TITLE
More pirate weapon content(two new cannon ammo types, a musket and flintlock resprite and rebalance) 

### PR DESCRIPTION
**Pull request by Boaz1111**
## About the PR

I added two new cannon ammo types(grapeshot and glassshot), gave the musket and flintlock a resprite and rebalance.
## Why / Balance

More pirate content, I'll probably make a PR making these things available to the crew in some way. 
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
image

## Breaking changes

**Changelog**

:cl:
- add: Added two new ammo types for the pirate cannon. Tally Ho!
- tweak: Changed the musket and flintlock sprites, as well as some functionality.

### CODE REVIEW

**Comment by IProduceWidgets** [comment]
Those cannon shots seem very powerful.
One thing you could do, but I don't really know if its better, is just fill the glass shot with the actual glass shard prototype from like broken lights or glasses and stuff. Anything that does throw damage will do damage out of a gun.

Does this bayonet on the musket work? I thought the reason it wasn't on the mosin was it no worky.
image

Does this make it always shoot 30 degrees off to the left or right?
image

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml [hunk 1]
```diff
@@ -69,36 +69,58 @@
...
+      path: /Audio/Weapons/bladeslice.ogg
+    
 - type: entity
   name: flintlock pistol
   parent: BaseWeaponSniper
   id: WeaponPistolFlintlock
   description: A pirate's companion. Yarrr! Uses .60 anti-materiel ammo.
   components:
+  - type: Gun
+    minAngle: 30
```
Comment (IProduceWidgets) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml:112]: Does this make it always shoot off 30 degrees left or right?

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml [hunk 2]
```diff
@@ -69,36 +69,58 @@
...
+  - type: UseDelay
+    delay: 8 #it's a musket
   - type: BallisticAmmoProvider
     whitelist:
       tags:
       - CartridgeAntiMateriel
     capacity: 1
     proto: CartridgeAntiMateriel
-
+  - type: MeleeWeapon
```
Comment (IProduceWidgets) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml:94]: Does this work? I thought the reason it wasn't on the mosin was that having a meleeweapon and gun component caused issues.

Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml:94]: I like the flintlock sprite, its a big improvement.

**Comment by Boaz1111** [comment]
> I like the flintlock sprite, its a big improvement.
> 
> 
> 
> Those cannon shots seem very powerful.
> 
> One thing you could do, but I don't really know if its better, is just fill the glass shot with the actual glass shard prototype from like broken lights or glasses and stuff. Anything that does throw damage will do damage out of a gun.

I wanted to have the shards stick into enemies for a bit

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml (hunk 1 above)
Comment (Boaz1111) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml:112]: nope, just a range

Comment (Boaz1111) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml:112]: I think
Fuck.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml (hunk 2 above)
Comment (Boaz1111) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml:94]: it works, I just had to make the musket fire on right click.

**Review by TheShuEd — APPROVED** [review]

**Comment by Tryded** [comment]
By the way, the use cooldown on the musket straight up doesn't work for guns after testing in game.

**Pull request merged**